Theming. Home page wip.

muresandavid12 opened the pull request.

muresandavid12 committed:
Theming. Home page wip.

paulrusu94 committed:
Started developing authentification pages.

- added basic layout
- started with sign in form page

paulrusustarchain reviewed (commented):
- public/banners/image21.png
- public/banners/image33.png
Da-le te rog un nume mai sugestiv, eventual ceva de genul: homepage-banner-1

- public/ellipse-social-bg.png -> Cred ca am putea obtine efectul asta folosind tailwindcss. Sa incercam sa pastram cat mai putine asseturi statice.

paulrusustarchain commented on src/app/(public)/page.tsx:
```diff
@@ -1,6 +1,53 @@
+                    <h1 className="mb-12 text-4xl text-gold font-medium">International<br />MEQUES Cup 2022</h1>
+                    <div className="text-white font-normal text-base">
+                        <p>Come celebrate with us and be the best jockey of the year</p>
+                        <p className="flex mt-3.5">Reward pool: <Image src={rewardIco} alt="Reward icon." className="ml-1 mr-2"/> 18,888</p>
```
As sugera sa creem o componenta `<Icon />` pe care sa o utilizam peste tot pentru a randa icons.

paulrusustarchain commented on src/app/(public)/layout.tsx:
```diff
@@ -7,9 +8,12 @@ export default function Layout({
 
     return <>
-        <div className="w-screen h-screen bg-homepage bg-no-repeat bg-cover">
+        <div className="w-screen h-screen flex-1 flex flex-col bg-homepage bg-no-repeat bg-cover">
```
Clasele de `flex` aici nu par sa schimbe cu nimic layoutul.

paulrusu94 committed:
Login and register forms/pages

- Added styles for login and register pages.
- Handled errors throw by authentication
- Created Input and Button ui components
- Added SSO with google.

muresandavid12 committed:
Homepage layout.

paulrusu94 committed:
Forgot password form.

- Created forgot password page.
- Implemented forgot password functionality using Firebase

paulrusustarchain commented on src/components/Sidebar.tsx:
```diff
@@ -1,18 +1,34 @@
+        </Link>
+        <Link className="flex mb-9 px-6"  href="/settings">
+            <Image src={marketplaceIco} alt="Marketplace icon." className="mr-1.5"/> Marketplace
+        </Link>
```
Lipseste butonul 'Daily Check-in'

paulrusustarchain commented on src/components/Sidebar.tsx:
```diff
@@ -1,18 +1,34 @@
-        <Link style={segment === 'dashboard' ? {textDecoration: 'underline'} : {}} className="py-4 pl-2 uppercase text-gray-800 font-light text-lg hover:text-black hover:bg-gray-200" href="/dashboard">Dashboard</Link>
-        <Link style={segment === 'settings' ? {textDecoration: 'underline'} : {}} className="py-4 pl-2 uppercase text-gray-800 font-light text-lg hover:text-black hover:bg-gray-200"  href="/settings">Settings</Link>
-
+    return <nav className="flex flex-col h-full w-1/6 px-6 justify-center bg-gray-900/30 font-medium text-sm text-gold ">
```
- width-ul sidebarului este dat de content width + paddings. Nu avem evoie de `w-1/6`.
- lipseste border-right effect.
- background color trebuie sa fie rgba(22,22,22,0.3)

paulrusustarchain commented on src/components/HomeSlider.tsx:
```diff
@@ -0,0 +1,44 @@
+            {slides.map((image: any, index: any) => (
+                <div
+                    key={index}
+                    className="absolute top-1/4 left-1/2 transform -translate-x-1/2 "
```
pozitionarea slideurilor nu e ok. sliderul trebuie sa fie pozitional la stanga, fara position absolute.

paulrusu94 committed:
Merge branch 'feature/authentification' into home-page

paulrusu94 committed:
Review fixes.

muresandavid12 committed:
Complete Quest slider.

muresandavid12 committed:
Show download flag.

muresandavid12 committed:
Slides centered.

paulrusu94 committed:
Updates for header and sidebar. Added Typography component.

paulrusu94 committed:
Merge branch 'home-page' of https://github.com/prorganization/meques-web into home-page

muresandavid12 committed:
Profile page WIP.

paulrusu94 committed:
Merge branch 'home-page' of https://github.com/prorganization/meques-web into home-page

paulrusu94 committed:
Added Metamask Button. Conect wallet feature.

paulrusu94 committed:
Added .env file.
